google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BuiltinMetricsTracerFactory.java

```diff
@@ -108,6 +110,12 @@ public static BuiltinMetricsTracerFactory create(
                 "The latency of the client application consuming available response data.")
             .setUnit(MILLISECOND)
             .build();
+    deadlineHistogram =
+        meter
+            .histogramBuilder(REMAINING_DEADLINE_NAME)
+            .setDescription("The customer specified operation deadline for the request.")
```

Comment:
This description is also not very accurate. We're recording the deadline when the request is sent to grpc. This will either be attempt timeout for the first few retries or the remaining deadline from operation timeout after retries and backoffs.